dev/integration_tests/ios_host_app/Host/AppDelegate.h

```diff
@@ -6,4 +6,8 @@
 #import <Flutter/Flutter.h>
 
 @interface AppDelegate : FlutterAppDelegate
+
+@property(nonatomic, strong) FlutterEngine* engine;
```

Comment:
This was moved from [dev/integration_tests/ios_add2app/ios_add2app/AppDelegate.h](https://github.com/flutter/flutter/pull/61007/files#diff-857228bf10c8c1e20cb51a974a25baa0) but git didn't follow the rename.